benchmarks/automated/tests/utils/apache_bench.py

```diff
@@ -147,6 +148,8 @@ def generate_csv_output(self, requests, concurrency, connection=None):
         with open(f"{self.local_tmp_dir}/result.txt") as f:
             data = f.readlines()
         artifacts["Benchmark"] = "AB"
+        artifacts["Batch Size"] = batch_size
+        artifacts["Mode"] = mode
```

Comment:
what does mode mean exactly?

Reply:
Here, mode is either 'eager_mod' or 'scripted_mode', just used for plotting it.

Reply:
I'd add a comment then to make it clearer

Reply:
Added